README.md

```diff
@@ -23,17 +23,34 @@ In short: JSON documents with named merkle-links that can be traversed.
 
 ## Repositories
 
-IPLD is a multifaceted, distributed effort. Here are our current repositories:
+IPLD is a multifaceted, distributed effort.
 
+### Specifications
+
+- [specs](https://github.com/ipld/specs) - The specifications for IPLD
 - [cid](https://github.com/ipld/cid) - Self-describing content-addressed identifiers for distributed systems
+
+### Implementations
+
+- [**JavaScript:** js-ipld-dag-cbor](https://github.com/ipld/js-ipld-dag-cbor) - JavaScript implementation of the IPLD spec.
```

Comment:
You can find documentation about implementation of _IPLD Formats_ at the interface-ipld-format https://github.com/ipld/interface-ipld-format

Another way is by using the seach in Github org - https://github.com/ipld?utf8=%E2%9C%93&q=js-ipld-&type=&language=